README.md

```diff
@@ -16,6 +16,21 @@ Just require katakata_irb or write it to your `.irbrc` file.
 require 'katakata_irb' rescue nil
 ```
 
+If you'd like to load types asynchronously, you can change the loader via `.irbrc`.
+
+```ruby
+# .irbrc
+begin
+  require 'katakata_irb'
+
+  # Change the loader type to asynchronous and start loading types on boot
+  KatakataIrb::Types.loader_type = :async
+  KatakataIrb::Types.preload
+rescue LoadError
+  nil
+end
+```
+
```

Comment:
Maybe I'll change the api, (I'm thinking to make async default. no configuration)
Can you exclude this README change?

Reply:
Done!